src/phalanx/control_system/update_shared_chart_version.py

```diff
@@ -0,0 +1,87 @@
+"""Script for updating shared chart versions."""
+
+import argparse
+import pathlib
+
+import yaml
+
+APPS_DIR = "applications"
+
+DIR_MAP = {"csc": "csc", "collector": "csc_collector"}
```

Comment:
I'm not really a fan of having two different names for the same thing and then doing a mapping. Can we find a way to consistently use the same name?

Reply:
This differentiation is gone in the phalanx cli implementation.